yarn.lock

```diff
@@ -2286,6 +2286,21 @@ dashdash@^1.12.0:
   dependencies:
     assert-plus "^1.0.0"
 
+datatables.net-dt@^1.10.20:
```

Comment:
Note for future reviewer - this file is here because I missed it in my PR

Reply:
NB: had to do this to get the rails server working:
`yarn install --check-files`

This warning was related but I don't know enough to worry about it:
`warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.`